src/ripple/websocket/Connection.h

```diff
@@ -262,8 +262,7 @@ Json::Value ConnectionImpl <WebSocket>::invokeCommand (Json::Value& jvRequest)
     Json::Value jvResult (Json::objectValue);
 
     auto required = RPC::roleRequired (jvRequest[jss::command].asString());
-    Role const role = requestRole (required, port_, jvRequest, m_remoteAddress,
-                                   getConfig().RPC_ADMIN_ALLOW);
+    Role const role = requestRole (required, port_, jvRequest, m_remoteAddress);
```

Comment:
Good catch, I'll change this to match the class style. Someone must have added port_ at a later date.